config/initializers/cors.rb

```diff
@@ -5,12 +5,12 @@
 
 # Read more: https://github.com/cyu/rack-cors
 
-# Rails.application.config.middleware.insert_before 0, Rack::Cors do
-#   allow do
-#     origins "example.com"
-#
-#     resource "*",
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+ Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins "http://localhost:3000"
+
+     resource "*",
+       headers: :any,
+       methods: [:get, :post, :put, :patch, :delete, :options, :head]
+   end
+ end
```

Comment:
- [ ] [optional] @DevKaranJ Great work implementing the changes requested, however due to different ports used to run the app, it might not in some instances, I suggest you replace `http://localhost:3000` to the initially suggested `*`.